tests/nlu/extractors/test_entity_synonyms.py

```diff
@@ -2,10 +2,17 @@
 from tests.nlu import utilities
```

Comment:
There is a file `tests/nlu/extractors/test_synonyms.py` with a test named `test_entity_synonyms` and there also exists a file named `test_entity_synonyms.py` . Can we consolidate them?